javascript/new-src/Gruntfile.js

```diff
@@ -124,7 +119,7 @@ module.exports = function(grunt) {
     cssmin: {
       build: {
         files: [{
-          src: 'css/style.css',
+          src: targets.css,
```

Comment:
From the documentation, this looks like it might not work properly if you add another file to the array:
https://github.com/gruntjs/grunt-contrib-cssmin#combine-two-files-into-one-output-file